source/standards/publish-opensource-code.html.md.erb

```diff
@@ -9,22 +9,22 @@ expires: 2018-09-01
 
 Wherever possible, we [make our source code open and reusable][]. This means other government departments and people in outside organisations can benefit from our work. We also maintain several open source projects developed for use on GOV.UK and with other work we do, such as the [Puppet module for go-audit][] or the [GDS Operations open source web site][].
 
-It’s not always appropriate to open code, there are situations [when you should keep some data and code closed][], for example when involving keys and security credentials.
+It’s not always appropriate to open code. There are situations [when you should keep some data and code closed][], for example when involving keys and security credentials.
 
 The Service Manual explains [how to open previously closed code and your responsibilities for maintaining open code][].
 
-You should release any open code with a clear license for reuse. As our documentation repositories include both content and code, we use 2 licences:
+You should release any open code with a clear license for reuse. As repositories often include both documentation and code, we use 2 licences:
 
 * [MIT licence][]  for code and code samples in documentation
 * [Open Government Licence (OGL)][] for documentation
 
 When you publish open source code, your project must:
 
-* include a readme, for example the [Digital Marketplace API][]
+* include a readme, for example [READMEs for GOV.UK applications][]
 * have useful and informative commit messages about why a change was made
 * provide a changelog, for example the [specification for CPAN Changes files][]
 * include an MIT and OGL licence file
-* link to a public list of known issues and bugs
+* link to a public list of known issues and bugs, for example [GOV.UK frontend toolkit][]
```

Comment:
this makes me wonder: should we explicitly state that we use github issues for our open source bug tracking, rather than being generic?

(this is outside the scope of this PR as it's a substantive change in meaning, not merely a copyedit. feel free to ignore this comment, @soupdragon99 -- I'll raise it with others)